exp/addons/api/v1beta1/clusterresourcesetbinding_types.go

```diff
@@ -129,6 +129,10 @@ type ClusterResourceSetBindingSpec struct {
 	// Bindings is a list of ClusterResourceSets and their resources.
 	// +optional
 	Bindings []*ResourceSetBinding `json:"bindings,omitempty"`
+
+	// ClusterName is the name of the Cluster this binding applies to.
+	// +optional
+	ClusterName string `json:"clusterName,omitempty"`
```

Comment:
I think this field should be a pointer to comply with the CAPI API conventions - https://cluster-api.sigs.k8s.io/contributing#optional-vs-required